matlab/msbuild/ice.proj

```diff
@@ -20,7 +20,7 @@
         </When>
         <Otherwise>
             <PropertyGroup>
-                <MatlabCommand>&quot;$(MatlabHome)\bin\matlab.exe&quot; -nodesktop -nosplash -wait -log -minimize</MatlabCommand>
+                <MatlabCommand>&quot;$(MatlabHome)\bin\matlab.exe&quot; -nodesktop -nosplash -wait -log -minimize -r</MatlabCommand>
```

Comment:
`-r` is required when building the toolbox locally.  It should not affect CI builds because we use the When above by setting `MATLAB_COMMAND`